proposals/vector_to_points.json

```
@@ -0,0 +1,68 @@
+{
+    "id": "vector_to_points",
+    "summary": "Sample random points from geometries",
+    "description": "Generate a vector data cube of points by sampling points from input geometries.",
+    "categories": [
+        "cubes",
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "Input geometries for sample extraction.\n\nTo maximize interoperability, a nested `GeometryCollection` should be avoided. Furthermore, a `GeometryCollection` composed of a single type of geometries should be avoided in favour of the corresponding multi-part type (e.g. `MultiPolygon`).",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "geojson"
+                },
+                {
+                    "type": "object",
+                    "subtype": "vector-cube"
+                }
+            ]
+        },
+        {
+            "name": "count",
```

As proposed in https://github.com/Open-EO/openeo-processes/pull/315#discussion_r782271205:

```suggestion
            "name": "size",
```